### ISSUE TITLE
Just the PDF for the slides

### ISSUE THREAD

**Issue by matrixise** [EuroPython/epcon#799]
The PDF file format is a standard.

**Comment by artcz** [comment]
How about jupyter notebooks?

**Comment by matrixise** [comment]
and pptx ? and keynote, etc... ?

**Comment by matrixise** [comment]
on speakerdeck, you share a pdf, not a notebook, keynote, etc...

**Comment by artcz** [comment]
True, but some people do presentations directly from the notebook, w/o any slides. It can be exported to PDF, but having notebook file is much nicer.

**Comment by matrixise** [comment]
@umgelurgel 2020?
